### PR TITLE
High-tier content config option

### PR DESCRIPTION
**Pull request by screret**

### CODE REVIEW

File: common/src/main/java/com/gregtechceu/gtceu/common/data/GTItems.java [hunk 1]
```diff
@@ -662,6 +662,55 @@ public static ICustomDescriptionId cellName() {
             .onRegister(compassNodeExist(GTCompassSections.COVERS, "pump", GTCompassNodes.COVER))
             .register();
 
+    public static ItemEntry<ComponentItem> ELECTRIC_PUMP_UHV = GTCEu.isHighTier() ? REGISTRATE.item("uhv_electric_pump", ComponentItem::create)
+            .lang("UHV Electric Pump")
+            .onRegister(attach(new CoverPlaceBehavior(GTCovers.PUMPS[8])))
+            .onRegister(attach(new TooltipBehavior(lines -> {
+                lines.add(Component.translatable("item.gtceu.electric.pump.tooltip"));
+                lines.add(Component.translatable("gtceu.universal.tooltip.fluid_transfer_rate", 1280 * 64 * 64 * 4 / 20));
```
Comment (mikerooni) [inline common/src/main/java/com/gregtechceu/gtceu/common/data/GTItems.java:670]: Transfer rates need to be adjusted in the tooltip

Comment (screret) [reply common/src/main/java/com/gregtechceu/gtceu/common/data/GTItems.java:670]: it's like this on 1.12 too, but sure, I'll do it

File: common/src/main/java/com/gregtechceu/gtceu/common/data/GTItems.java [hunk 2]
```diff
@@ -798,10 +847,55 @@ public static ICustomDescriptionId cellName() {
...
+                lines.add(Component.translatable("gtceu.universal.tooltip.item_transfer_rate_stacks", 16));
             })))
             .onRegister(compassNodeExist(GTCompassSections.COVERS, "conveyor", GTCompassNodes.COVER))
             .register();
+    public static ItemEntry<ComponentItem> CONVEYOR_MODULE_UHV = GTCEu.isHighTier() ? REGISTRATE.item("uhv_conveyor_module", ComponentItem::create)
+            .lang("UHV Conveyor Module")
+            .onRegister(attach(new CoverPlaceBehavior(GTCovers.CONVEYORS[8])))
+            .onRegister(attach(new TooltipBehavior(lines -> {
+                lines.add(Component.translatable("item.gtceu.conveyor.module.tooltip"));
+                lines.add(Component.translatable("gtceu.universal.tooltip.item_transfer_rate_stacks", 16));
```
Comment (mikerooni) [inline common/src/main/java/com/gregtechceu/gtceu/common/data/GTItems.java:899]: Transfer rate needs to be adjusted in tooltips

File: common/src/main/java/com/gregtechceu/gtceu/common/data/GTItems.java [hunk 3]
```diff
@@ -707,7 +707,7 @@ public static ICustomDescriptionId cellName() {
             .onRegister(attach(new CoverPlaceBehavior(GTCovers.PUMPS[12])))
             .onRegister(attach(new TooltipBehavior(lines -> {
                 lines.add(Component.translatable("item.gtceu.electric.pump.tooltip"));
-                lines.add(Component.translatable("gtceu.universal.tooltip.fluid_transfer_rate", 1280 * 64 * 64 * 4 / 20));
+                lines.add(Component.translatable("gtceu.universal.tooltip.fluid_transfer_rate", 1280 * 64 * 64 * 64 * 64 / 20));
```
Comment (mikerooni) [inline common/src/main/java/com/gregtechceu/gtceu/common/data/GTItems.java]: I can't add an inline suggestion for some reason, but probably needs `1280L` here too

**Review by mikerooni — COMMENTED** [review]
Looks good so far. Still needs high tier fluid regulators

**Review by mikerooni — APPROVED** [review]

**Pull request merged**